### PR TITLE
Fix the problem of self.original_text index in class Text

### PR DESCRIPTION
**Pull request by xia0long**
## Overview: What does this pull request change?

+ Fix doc in function `Text`.
+ use `self.original_text` or `self.text` depending on the situation.

## Motivation and Explanation: Why and how do your changes improve the library?

Bugfix.

## Reviewer Checklist
- [ ] The PR title is descriptive enough for the changelog, and the PR is labeled correctly
- [ ] If applicable: newly added non-private functions and classes have a docstring including a short summary and a PARAMETERS section
- [ ] If applicable: newly added functions and classes are tested

### CODE REVIEW

File: manim/mobject/svg/text_mobject.py [hunk 1]
```diff
@@ -373,7 +373,7 @@ def construct(self):
                 arb = Text(
                     "صباح الخير \n تشرفت بمقابلتك", font="sans-serif"
                 )  # don't mix RTL and LTR languages nothing shows up then ;-)
-                japanese = Text("臂猿「黛比」帶著孩子", font="sans-serif")
+                chinese = Text("臂猿「黛比」帶著孩子", font="sans-serif")
                 self.add(morning,chin,mess,russ,hin,arb,japanese)
```
Comment (behackl) [inline manim/mobject/svg/text_mobject.py:377]: Yikes, I didn't notice we had Chinese and Japanese swapped in the library.

```suggestion
                self.add(morning, chinese, mess, russ, hin, arb, japanese)
```

**Review by behackl — CHANGES_REQUESTED** [review]
I've tested these changes locally (we need to think about how we can start adding Text tests; we might need to include our own font files...) – anyhow, while this fixes the behavior for `disable_ligatures=False`, it breaks `disable_ligatures=True`.

My test scene:

```py
class Te(Scene):
    def construct(self):
        t = Text("Cherry Blossoms in Spring", 
            disable_ligatures=True, 
            t2c={"Cherry": RED, "Spring": GREEN}
        ).scale(2)

        self.add(t)
```

**Comment by behackl** [comment]
-- and of course: thank you for your contribution!

**Comment by xia0long** [comment]
Or do we need another mechanism to set the color of multiple characters? no need self.text,  just find the indexes of characters in  `t2g`, it supports the string with blank and simple.

**Comment by xia0long** [comment]
@behackl Thanks for your reply! I'll do some tests and fix this problem.

**Pull request closed without merging**